.github/workflows/publish-nuget.yml

```diff
@@ -0,0 +1,12 @@
+name: Publish to NuGet
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  call-publish-workflow:
+    uses: Lombiq/GitHub-Actions/.github/workflows/publish-nuget.yml@dev
+    secrets:
+      API_KEY: ${{ secrets.DEFAULT_NUGET_PUBLISH_API_KEY }}
```

Comment:
@Piedone Is this secret available on the repo?

Also iirc this will only work after it's merged into dev, right?

Reply:
It's an organizational secret so yes, it's available. I'm not sure whether it can trigger already, I think it will, but you can try.

Do you [the template](https://github.com/Lombiq/Open-source-template/blob/dev/.github/workflows/publish-nuget.yml) though.

Reply:
> Do use [the template](https://github.com/Lombiq/Open-source-template/blob/dev/.github/workflows/publish-nuget.yml) though.

So close!